### PR TITLE
Change 01_linear_regression.py for new version of TensorFlow

### PR DESCRIPTION
**Pull request by daikikojima**
When I run 01_linear_regression.py in my pc, 
terminal says 
"WARNING:tensorflow:From 01_linear_regression.py:27 in <module>.: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead."
So, I changed tf.initialize_all_variables().run() to tf.global_variables_initializer().run()

### CODE REVIEW

**Comment by hunkim** [comment]
Looks good. Can you also midify the .yml file to install the latest version of TF?

**Comment by daikikojima** [comment]
I fixed .yml.

File: .travis.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ install:
   - pip install matplotlib
   # install TensorFlow from https://storage.googleapis.com/tensorflow/
   - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.9.0-cp27-none-linux_x86_64.whl;
+      pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.12.1-cp27-none-linux_x86_64.whl;
```
Comment (hunkim) [inline .travis.yml]: Perhaps just `php install tensorflow` for the automatic future update?

**Comment by hunkim** [comment]
@daikikojima TF 1.0 alpha came out. Do you think you can update code using 1.0 alpha?

**Comment by hunkim** [comment]
Please check out #69

**Pull request closed without merging**